RUFAS/biophysical/manure/storage/composting.py

```diff
@@ -0,0 +1,553 @@
+            elif 15 <= manure_temperature <= MCF_UPPER_BOUND_TEMPERATURE:
+                return MCF_COMPOSTING_WINDROW_MEDIUM
+            else:
+                return MCF_COMPOSTING_WINDROW_HIGH
+
+    @staticmethod
+    def _calculate_dry_matter_loss(methane_emission: float, carbon_decomposition: float) -> float:
+        """
+        This function calculates the total dry matter loss of the current day.
+
+        Returns
+        -------
+        float
+            The total dry matter loss of the current day, kg/day.
+        """
+        return 2 * carbon_decomposition + methane_emission
+
+    @staticmethod
+    def _calculate_carbon_decomposition(
+        manure_temperature: float, non_degradable_volatile_solids: float, degradable_volatile_solids: float
+    ) -> float:
+        """
+        This function calculates the total carbon decomposition of the current day.
+
+        Parameters
+        ----------
+        manure_temperature : float
+            The manure temperature of the current day, Celsius.
+        non_degradable_volatile_solids : float
+            The non-degradable volatile solids of the current day, kg.
+        degradable_volatile_solids : float
+            The degradable volatile solids of the current day, kg.
+
+        Returns
+        -------
+        float
+            The total carbon decomposition of the current day, kg/day.
+        """
+        carbon_decomposition_rate = Composting._calculate_carbon_decomposition_rate(manure_temperature)
+        anaerobic_coefficient = Composting._calculate_anaerobic_coefficient()
+
+        return (
+            (
+                degradable_volatile_solids * DEFAULT_CARBON_FRACTION_AVAILABLE_IN_VSD
+                + non_degradable_volatile_solids * DEFAULT_CARBON_FRACTION_AVAILABLE_IN_VSND
+            )
+            * carbon_decomposition_rate
+            * DEFAULT_EFFECT_OF_MOISTURE_ON_MICROBIAL_DECOMPOSITION
+            * anaerobic_coefficient
+        )
+
+    @staticmethod
+    def _calculate_carbon_decomposition_rate(manure_temperature: float) -> float:
+        """
+        This function calculates the carbon decomposition rate of the current day.
+
+        Parameters
+        ----------
+        manure_temperature : float
+            The manure temperature of the current day, Celsius.
+
+        Returns
+        -------
+        float
+            The carbon decomposition rate of the current day, per day.
+        """
+        max_microbial_decomposition_rate = Composting._calculate_max_microbial_decomposition_rate()
+        slow_microbial_decomposition_rate = Composting._calculate_slow_fraction_decomposition_rate(manure_temperature)
+
+        return float(
+            (
+                (max_microbial_decomposition_rate - slow_microbial_decomposition_rate)
+                * (math.e ** (FIRST_ORDER_DECAYING_COEFFICIENT * (DEFAULT_DAYS_SINCE_LAST_TURNING - DEFAULT_LAG_TIME)))
+                * slow_microbial_decomposition_rate
```

Comment:
```suggestion
                + slow_microbial_decomposition_rate
```
This is an error from the original module, fixing it here for the same reasons as above - we're already modifying C decomposition values so might as well fix things if values will be different regardless.
![image](https://github.com/user-attachments/assets/ae8a2cbe-92f8-47e5-9b0b-a0df831a9175)